locations/spiders/olivegarden.py

```diff
@@ -15,6 +15,9 @@ class OliveGardenSpider(SitemapSpider):
         "https://www.olivegarden.com/en-locations-sitemap.xml",
     ]
     sitemap_rules = [(r"[0-9]+$", "parse")]
+    custom_settings = {
+        "ZYTE_API_TRANSPARENT_MODE": True,
```

Comment:
If this could be enabled as a middleware it could be honour `requires_proxy`. I'll take a look at this.

cc @jimbofreedman https://github.com/alltheplaces/alltheplaces/compare/master...huq-industries:atp:master#diff-a491ec303137ac44419b40d880e6edd634035ecd557f21499eec8c76365c92fe

Reply:
That's quite a big diff - what would you like my attention on 😁

Reply:
Haha, I was trying to link to your [SmartProxyBridgeMiddleware](https://github.com/huq-industries/atp/blob/f7b20dc668303bc8f0784f779dd13604d8fdaa73/locations/middlewares/smartproxy_bridge.py).

Reply:
OK, confusingly you guys are using different services, and different packages. [scrapy-zyte-smartproxy](https://docs.zyte.com/smart-proxy-manager-get-started.html) and [scrapy-zyte-api](https://docs.zyte.com/zyte-api/get-started.html). FYI @jimbofreedman yours is labelled as "Legacy".

I don't think using 2 services will be that bad, as long as we use the same data to decide when to proxy. (`requires_proxy` vs `ZYTE_API_TRANSPARENT_MODE`)

Reply:
Oh cool - yeah we've managed to integrate with different Zyte products. API looks like an upgrade so I'll try and give it a whirl :D

Reply:
> OK, confusingly you guys are using different services

If the numbers are better on API, which I expect they will be, we'll move over.

Reply:
There's a tiny blurb saying that Zyte API is meant to replace Smart Proxy Manager on [this page](https://docs.zyte.com/smart-proxy-manager-get-started.html), so I went straight to the API product. It also seems to be a bit cheaper and has  a $0/month minimum with pay-as-you-go and a spending limit, which is pretty great for All the Places.